hack/make-docker-images.sh

```diff
@@ -37,15 +37,15 @@ while IFS= read -d $'\0' -r dir; do
     (
         cd "${builddir}"
         log "Building: ${image}"
-        docker build -t "${image}" .
+        docker build --pull -t "${image}" .
 
         log "Pushing: ${image}"
         docker push "${image}"
 
         if [ $svcname != "frontend" ] && [ $svcname != "loadgenerator" ]
         then
             log "Building: ${image}-native-grpc-probes"
-            docker build -t "${image}-native-grpc-probes" . --target without-grpc-health-probe-bin
+            docker build --pull -t "${image}-native-grpc-probes" . --target without-grpc-health-probe-bin
             log "Pushing: ${image}-native-grpc-probes"
             docker push "${image}-native-grpc-probes"
         fi
```

Comment:
**Context:**
* This `hack/make-docker-images.sh` script is run whenever we create a new release of Online Boutique.
* It builds (and pushes) the [images at gcr.io/google-samples/microservices-demo](https://pantheon.corp.google.com/gcr/images/google-samples/global/microservices-demo?mods=prod_coliseum&project=google-samples).